addons/website_customer/controllers/main.py

```diff
@@ -118,6 +118,8 @@ def customers(self, country=None, industry=None, page=0, **post):
         google_map_partner_ids = ','.join(str(it) for it in partners.ids)
         google_maps_api_key = request.website.google_maps_api_key
 
+        # TODO: The website published condition should be removed, an ACL on website_published should be added
```

Comment:
We could ignore this, that's a minor feature, but that's not common to hide unpublished stuff from the admin.
If we want it to fit what we do elsewhere, we should do what I wrote in comment IMO

Reply:
:+1: